src/main/kotlin/no/nav/syfo/client/OppgaveClient.kt

```diff
@@ -64,6 +61,26 @@ class OppgaveClient constructor(val url: String, val oidcClient: StsOidcClient)
                 prioritet = "NORM"
         )
 
-        return createOppgave(opprettOppgave, sykmeldingId)
+        return opprettOppgave(opprettOppgaveRequest, sykmeldingId).id
     }
 }
+
+data class OpprettOppgaveRequest(
+    val tildeltEnhetsnr: String? = null,
```

Comment:
Er alle disse optional? Eller er det for å slippe litt billigere unna i test?

Reply:
Det er kun aktivDato og prioritet som er påkrevd av api'et, så da tenkte jeg det var greit at requesten speilet det :) 

https://oppgave.nais.preprod.local/?url=https://oppgave.nais.preprod.local/api/swagger.json#/v1oppgaver/opprettOppgave